index.html

```diff
@@ -50,6 +138,37 @@
                     </li>
                 </ul>
             </div>
+            <button class="work-link">Get my resume</button>
+            <ul class="skills">
+                <li>
+                    <div class="languages">
+                        <p>Languages</p>
+                        <img src="./assets//images/arrow-down.svg" alt="" srcset="">
+                    </div>
+                    <ul class="languages-list">
+                        <li>
+                            <img src="./assets/images/js.svg" alt="" srcset="">
+                            <p>Javascript</p>
+                        </li>
+                        <li>
+                            <img src="./assets/images/html.svg" alt="" srcset="">
+                            <p>HTML</p>
+                        </li>
+                        <li>
+                            <img src="./assets/images/css.svg" alt="" srcset="">
+                            <p>CSS</p>
+                        </li>
+                    </ul>
+                </li>
+                <li class="flex-list">
+                   <p>Frameworks</p>
```

Comment:
Let's add this `chevron` icon to stick to the Figma design 👍

| Your design | Figma design |
| :---: | :---: |
| ![Capture](https://user-images.githubusercontent.com/80458199/236184701-2258b8c0-ee84-4845-b183-f76ce1464b92.PNG) | ![Capture 1](https://user-images.githubusercontent.com/80458199/236184742-72f47b20-e9ff-4231-8d04-764d60001762.PNG) |